src/lib/y2network/config_writer/sysconfig.rb

```diff
@@ -31,7 +31,7 @@ class Sysconfig
       #
       # @param config [Y2Network::Config] Configuration to write
       def write(config)
-        routes = config.routes.map { |r| route_to_hash(r) }
+        routes = config.routing.routes.map { |r| route_to_hash(r) }
```

Comment:
route_to_hash writes default route as "-" but we only support import with "default"

Reply:
will fix when merging scr write PR